playbooks/templates/.env.j2

```diff
@@ -176,19 +176,18 @@ SKYNET_ACCOUNTS_LOG_LEVEL={{ webportal_common_config.accounts_log_level }}
 # - 'false': All users incl. anonymous users can access the portal (access to portal is public)
 # REQUIRED: manually defined by user, or default is set in docker-compose file
 ACCOUNTS_LIMIT_ACCESS={{ webportal_common_config.accounts_limit_access | string | lower }}
+{% endif %}
 
-{% if webportal_common_config.accounts_limit_access | string | lower != 'false' %}
```

Comment:
Are `ACCOUNTS_TEST_USER_...` vars required when `ACCOUNTS_LIMIT_ACCESS=false`, i.e. on anonymous portal? I thing they are not required in this case and should be enclosed in `{% if %}` block.

Reply:
yes I think you're right, I think this if statement should stay. The original issue was that this if statement shouldn't be enclosed in the outer if statement because `account_limit_access` can be `undefined` and these values are needed.

I'll update. Good catch.